releng/org.eclipse.xtext.contributor/Xtext.setup

```diff
@@ -616,7 +619,40 @@
         <repository
             url="${p2.buildship}"/>
         <repository
-            url="${p2.webtools}/2022-12"/>
+            url="${p2.webtools}/2023-03"/>
+        <repository
+            url="${p2.lsp4j}"/>
+        <repository
+            url="${p2.cbi.analyzer}"/>
+        <repository
+            url="${p2.egit}"/>
+        <repository
+            url="${p2.license}"/>
+      </repositoryList>
+      <repositoryList
+          name="2023-06">
+        <repository
+            url="https://download.eclipse.org/eclipse/updates/4.28-I-builds"/>
+        <repository
+            url="${p2.mwe2}"/>
+        <repository
+            url="${p2.antlr-gen}"/>
+        <repository
+            url="${p2.m2e}/2.2.1"/>
+        <repository
+            url="${p2.swtbot}"/>
+        <repository
+            url="${p2.orbit}"/>
+        <repository
+            url="${p2.emf}"/>
+        <repository
+            url="${p2.gef-classic}"/>
+        <repository
+            url="${p2.xpand}"/>
+        <repository
+            url="${p2.buildship}"/>
+        <repository
+            url="${p2.webtools}/2023-03"/>
```

Comment:
Is that intentional? Because there's no 2023-06 for that?

Reply:
i wanted to wait for m1 out to change it